### PR TITLE
Add HapticType

### PR DESCRIPTION
**Pull request by lidord-wix**
## Description
Export enum HapticType instead of hapticMethods

## Changelog
Export enum HapticType instead of hapticMethods

### CODE REVIEW

File: src/services/HapticService.ts [hunk 1]
```diff
@@ -24,6 +24,6 @@ function triggerHaptic(hapticMethod: string, componentName: string) {
 }
 
 export default {
-  HapticMethods,
+  HapticType,
```
Comment (mendyEdri) [inline src/services/HapticService.ts:27]: No migration needed?

File: src/services/HapticService.ts [hunk 2]
```diff
@@ -5,17 +5,17 @@ const options = {
...
+  impactLight = 'impactLight',
+  impactMedium = 'impactMedium',
+  impactHeavy = 'impactHeavy',
+  notificationSuccess = 'notificationSuccess',
+  notificationWarning = 'notificationWarning',
+  notificationError = 'notificationError'
+}
 
-function triggerHaptic(hapticMethod: string, componentName: string) {
+function triggerHaptic(hapticMethod: HapticType, componentName: string) {
```
Comment (mendyEdri) [inline src/services/HapticService.ts:18]: It's intentionally kept its name `method` where all other places has changed to `type`?

**Review by mendyEdri — COMMENTED** [review]
Minor changes :)

File: src/services/HapticService.ts (hunk 1 above)
Comment (lidord-wix) [reply src/services/HapticService.ts:27]: Nope, it's a new code so it's not in use yet :)

File: src/services/HapticService.ts (hunk 2 above)
Comment (lidord-wix) [reply src/services/HapticService.ts]: It was intentional, but on second thought it is better to be aligned with our 'old' code in the private repo.

**Review by mendyEdri — APPROVED** [review]

**Pull request merged**